README.md

```diff
@@ -147,6 +148,18 @@ A job can be inserted with optional params:
 |> MyApp.Repo.insert()
 ```
 
+When using `:idempotency_key` is recomended configure the Repo options `on_conflict: :nothing, conflict_target: [:idempotency_key]` to do not throw erros from database:
+
+```elixir
+%{"type" => "SendEmail", "address" => "jonas@gmail.com", "body" => "Welcome!"}
+|> MyApp.JobQueue.new(priority: 2, max_attempts: 2, idempotency_key: "Welcome!")
```

Comment:
Could we use a slightly more realistic example, where the idempotency key for an email would be some combination of address and subject?